src/steps/owners/converters.ts

```diff
@@ -1,19 +1,23 @@
 import {
+  createDirectRelationship,
   createIntegrationEntity,
+  Entity,
   parseTimePropertyValue,
+  Relationship,
+  RelationshipClass,
 } from '@jupiterone/integration-sdk-core';
 import { Owner } from '../../types';
 import { getEntityKey } from '../../utils';
-import { Entities } from '../constants';
+import { Entities, Relationships } from '../constants';
 
 export function createOwnerEntity(data: Owner) {
   return createIntegrationEntity({
     entityData: {
       source: data,
       assign: {
-        _class: Entities.USER._class,
-        _type: Entities.USER._type,
-        _key: getEntityKey(Entities.USER, data.id),
+        _class: Entities.OWNER._class,
+        _type: Entities.OWNER._type,
+        _key: getEntityKey(Entities.OWNER, data.id),
```

Comment:
We might want to add `firstName` and `lastName` as additional properties as well, if you recall that comment in the WPEngine :slightly_smiling_face: